src/components/ExerciseHeader.tsx

```diff
@@ -73,12 +73,16 @@ const ExerciseHeader = ({
       : getLabels().general.header.cancelExercise
 
   useEffect(() => {
+    const handleOnPress = () => {
+      if (!confirmClose && isLastWordInList) {
```

Comment:
I'm not sure we still need the `!confirmClose`. Can you explain its role here to me?

Reply:
Thank you @LeandraH for your review and question. I also think it does nothing here. I think because of this previous line of code was not working properly. I think I can convert my changes into one line of code like it was before only replacing `!confirmClose` with `isLastWordInList`

Reply:
Hmmm, no, when you look at the props of the ExerciseHeader, we assign the default `confirmClose = true` in line 58.

Before this PR, there were two possible cases here of what happens after a user taps on the close button: Either we ask if the user is sure that they want to close the exercise, or it closes immediately. In both cases, the user is sent to the overview of the levels, either straight away or after confirming. What we want to add in this PR is a third scenario where the user is not sent to the overview of the levels but to the word list. So we need to add a possible outcome, not replace one.